test/activities/ActivityUsageEntity.js

```diff
@@ -0,0 +1,258 @@
+import { ActivityUsageEntity } from '../../src/activities/ActivityUsageEntity.js';
+import { testData } from './data/ActivityUsageEntity.js';
+
+describe('ActivityUsageEntity', () => {
+	let entity, readonlyEntity;
+
+	beforeEach(() => {
+		const entityJson = window.D2L.Hypermedia.Siren.Parse(testData.activityUsageEntityEditable);
+		entity = new ActivityUsageEntity(entityJson);
+
+		const readonlyJson = window.D2L.Hypermedia.Siren.Parse(testData.activityUsageEntityReadOnly);
+		readonlyEntity = new ActivityUsageEntity(readonlyJson);
+	});
+
+	describe('Basic loading', () => {
+		it('can get organizationHref', () => {
+			expect(entity.organizationHref()).to.equal('http://vlx1-mdulat.desire2learn.d2l:44444/d2l/api/hm/organizations/6609');
+		});
+
+		it('can get specializationHref', () => {
+			expect(entity.specializationHref()).to.equal('http://vlx1-mdulat.desire2learn.d2l:44444/d2l/api/hm/assignments/6609/folders/31');
+		});
+
+		it('can get userActivityUsageHref', () => {
+			expect(entity.userActivityUsageHref()).to.equal('http://vlx1-mdulat.desire2learn.d2l:44444/d2l/api/hm/activities/activities/6606_2000_31/usages/6609/users/169');
+		});
+
+		it('can edit release conditions', () => {
+			expect(entity.canEditReleaseConditions()).to.be.true;
+		});
+
+		it('can get release conditions url', () => {
+			expect(entity.editReleaseConditionsUrl()).to.equal('/d2l/le/conditionalRelease/6609/dialog/dropboxes/31/openDialog');
+		});
+	});
+
+	describe('Due Date', () => {
+		describe('Can edit', () => {
+			let sandbox, setDueDateSpy;
```

Comment:
minor thing, the `sandbox` can be defined for the whole file and created/restored in the topmost `beforeEach/afterEach`. Just removes a bit of repeated setup! (albeit not much, admittedly)